package.json

```diff
@@ -37,7 +37,8 @@
     "framer-motion": "^10.15.0",
     "next": "13.4.9",
     "react-player": "^2.13.0",
-    "storyblok-rich-text-react-renderer-ts": "^3.2.0"
+    "storyblok-rich-text-react-renderer-ts": "^3.2.0",
+    "usehooks-ts": "^2.9.1"
```

Comment:
Installed this package since it has some useful hooks which I might need later. I'm using the copy to clipboard one for this PR

Reply:
I like this package.